frontends/concrete-python/concrete/fhe/compilation/compiler.py

```diff
@@ -447,14 +447,6 @@ def compile(
             self._evaluate("Compiling", inputset)
             assert self.graph is not None
 
-            if len(self.graph.output_nodes) > 1:
-                fmtd_graph = self.graph.format(
-                    highlighted_result=["multiple outputs are not supported"],
-                    show_bounds=False,
-                )
-                message = "Function you are trying to compile cannot be compiled\n\n" + fmtd_graph
-                raise RuntimeError(message)
-
```

Comment:
There should be a test for this somewhere, you need to remove that I think.

Reply:
done